operations/configuration_management/NCN_Worker_Image_Customization.md

```diff
@@ -0,0 +1,247 @@
+# NCN Worker Image Customization
+
+**NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
+is only relevant for booting compute nodes and can be ignored when working with NCN images.
+
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify
+a Ceph image.
+
+1. Locate the NCN image to be modified.
```

Comment:
This step says to locate the image to be modified, but doesn't give instructions on how to locate it that I can see. How should the user determine the value for ARTIFACT_VERSION?

Reply:
It depends what they're trying to do or whether it's the first time modifying the image. It's difficult to be prescriptive here so it being vague is somewhat intentional. I can provide an example scenario where the assumption is they want to modify the image that is currently running, but that is not always the case. It could be they want to modify and image during an upgrade prior to it ever booting.